### ISSUE TITLE
Evaluate if the process worked?

### ISSUE THREAD

**Issue by ff6347** [technologiestiftung/exhibition-remote-control#1]
From the docs I see that `plug.setPowerState()` returns a `Promise <true>` would be good not only to assume the plug is on, but to use the response. ;)
https://github.com/technologiestiftung/exhibition-remote-control/blob/c0ee643af4ede9e8695de5715ca4a124c8399ca9/index.js#L43

**Comment by fabianmoronzirfas** [comment]
No. I mean the plug could return false and you wouldn't notice. Just normal error handling

**Comment by jvolker** [comment]
What would you recommend in such a case? Throw an error in the UI?

**Comment by fabianmoronzirfas** [comment]
Yes. Since no one will watch the shell.

**Comment by jvolker** [comment]
But they should watch the exhibition. If something is not working they are going to complain anyway. ;)

**Comment by fabianmoronzirfas** [comment]
But an error handler that tells you on screen "hey. exhibit foo could not start the power" is a nice feature. If this would be there the user knows that the plug didn't or did work. 

IMO it helps troubleshooting the process.
